app/org/sagebionetworks/bridge/dynamodb/DynamoIndexHelper.java

```diff
@@ -63,9 +67,31 @@ public void setMapper(DynamoDBMapper mapper) {
         }
         return recordKeyList;
     }
-
+    
     /**
-     * Queries the global secondary index with the specified key name and value. Results will be returned as a list of
+     * Query via a secondary index to return the count of matching items in the table.
+     *  
+     * @param indexKeyName
+     *         index key name to query on
+     * @param indexKeyValue
+     *         index key value to query on
+     * @param rangeKeyCondition
+     *         range condition for query on range portion of key (optional)
+     * @return count of records in the table
+     */
+    @SuppressWarnings("unused")
```

Comment:
I don't think we should ever suppress this warning. If we're not using it and we're not testing it, why have it?